libpod/image/search.go

```diff
@@ -64,11 +64,11 @@ type SearchFilter struct {
 // SearchImages searches images based on term and the specified SearchOptions
 // in all registries.
 func SearchImages(term string, options SearchOptions) ([]SearchResult, error) {
-	// Check if search term has a registry in it
-	registry, err := sysreg.GetRegistry(term)
-	if err != nil {
-		return nil, errors.Wrapf(err, "error getting registry from %q", term)
+	registry := ""
```

Comment:
@mtrmac, does that reflect what you had in mind?

Reply:
Yes, exactly (but I haven’t done the legwork to check that there are enough tests for the relevant input formats to know I’m not missing a corner case).